src/lib.rs

```diff
@@ -124,31 +135,6 @@ impl From<CaptureIndexError> for Error {
     }
 }
 
-#[derive(Default)]
```

Comment:
I got to track down the fact that sometimes I wasn't seeing a newly added lint-rule violation get printed out

The reason ended up being that I forgot that in adding `query_per_language` (to `Args`) then the capture index could no longer be a "singleton" (because now each language's query could have a different "target" capture index)

So got rid of this data structure for it entirely and moved the per-language-query capture index storage/calculation into `CachedQueries`